src/controllers/AnnotationModeController.js

```diff
@@ -151,7 +151,7 @@ class AnnotationModeController extends EventEmitter {
      */
     getButton(annotatorSelector: string): HTMLElement {
         // $FlowFixMe
-        return this.container.querySelector(annotatorSelector);
+        return this.headerElement.querySelector(annotatorSelector);
```

Comment:
This will always exist. See https://github.com/box/box-annotations/pull/233#discussion_r222101588